src/Console/Shell.php

```diff
@@ -163,6 +163,7 @@ public function __construct(ConsoleIo $io = null)
 
         $locator = $this->tableLocator() ? : 'Cake\ORM\TableRegistry';
         $this->modelFactory('Table', [$locator, 'get']);
+        $this->modelType('Table');
```

Comment:
Needing to call this method makes this change backwards incompatible. Including the trait no longer gives you access to the ORM locator anymore by default - which it probably should to be more backwards compatible.